packages/apps/reputation-oracle/server/src/database/migrations/1705545284387-AddKYCColumnsToUser.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddKYCColumnsToUser1705545284387 implements MigrationInterface {
+  name = 'AddKYCColumnsToUser1705545284387';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
I think is better to have KYC info in its own table instead of adding columns to Users table